open.md

```diff
@@ -0,0 +1,28 @@
+---
+layout: single-page-md
+title: Open Contest
+key: open
+---
+
+We invite everybody to participate in APIO 2020 Open Contest.
```

Comment:
Tambah semacam: we politely ask contestants from APIO 2020 to not participate in this contest ... ?

Reply:
Yang ini gw renungkan dulu :)

Reply:
Added based on discussion on fb.